### PR TITLE
feat(lol): new function `qrcode`

### PR DESCRIPTION
**Pull request by tinogomes**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- `qrcode` generates a QRCode image. Requires internet access

### CODE REVIEW

File: plugins/lol/lol.plugin.zsh [hunk 1]
```diff
@@ -49,3 +49,14 @@ alias yolo='git commit -m "$(curl -s http://whatthecommit.com/index.txt)"'
...
+qrcode() {
+  url=$1
+
+  if [[ "$url" != http* ]]; then
+    url=https://$1
+    echo
+    echo QRCode for: $url
+  fi
+
+  curl qrcode.show/$url
```
Comment (JuniorJPDJ) [inline plugins/lol/lol.plugin.zsh:61]:
```suggestion
  curl qrcode.show -d "$url"
```

**Comment by JuniorJPDJ** [comment]
You sometimes want to qrcode something which is not url, I'd not do protocol logic here.

**Comment by mcornella** [comment]
Does this really fit with the intent of the `lol` plugin? Maybe this would go better as its own plugin, with the functions defined in the [SHELL FUNCTIONS section](https://qrcode.show/).

**Comment by JuniorJPDJ** [comment]
I also suggest writing to qrcode.show admins about adding support to ohmyzsh. They could probably add info to their main page.

**Comment by tinogomes** [comment]
> I also suggest writing to qrcode.show admins about adding support to ohmyzsh. They could probably add info to their main page.

I will try to do that too.

**Comment by tinogomes** [comment]
extract to a new plugin #10239

**Pull request closed without merging**